src/components/mixins/InputSliderMixin.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Vue } from 'vue-property-decorator';
+
+import { mutation } from '@/store/decorators';
+import { FocusedField } from '@/store/removeLiquidity/types';
+
+@Component
+export default class InputSliderMixin extends Vue {
+  @mutation.removeLiquidity.setFocusedField setFocusedField!: (field: FocusedField) => void;
```

Comment:
Binding to removeLiqudity module shouldn't be in reusable mixin.
Please add event listener in parent components on `focus `event